filter-core/src/main/java/org/wso2/micro/gateway/filter/core/constants/APIConstants.java

```diff
@@ -149,4 +153,49 @@ private SubscriptionValidationResources() {
 
         }
     }
+
+    /**
+     * Supported event types
+     */
+    public enum EventType {
+        API_CREATE,
+        API_UPDATE,
+        API_DELETE,
+        API_LIFECYCLE_CHANGE,
+        APPLICATION_CREATE,
+        APPLICATION_UPDATE,
+        APPLICATION_DELETE,
+        APPLICATION_REGISTRATION_CREATE,
+        POLICY_CREATE,
+        POLICY_UPDATE,
+        POLICY_DELETE,
+        SUBSCRIPTIONS_CREATE,
+        SUBSCRIPTIONS_UPDATE,
+        SUBSCRIPTIONS_DELETE,
+        SCOPE_CREATE,
+        SCOPE_UPDATE,
+        SCOPE_DELETE
+    }
+
+    /**
+     * Supported policy types
+     */
+    public enum PolicyType {
+        API,
+        APPLICATION,
+        SUBSCRIPTION
+    }
+
+    /**
+     * Topic Names
```

Comment:
possible check style failure 
```suggestion
     * Topic Names.
```